fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/data/SavingsAccountTransactionData.java

```diff
@@ -155,7 +155,7 @@ public boolean isDebit() {
     }
 
     public boolean isWithdrawalFeeAndNotReversed() {
-        return this.transactionType.isWithdrawalFee() && isNotReversed();
+        return this.transactionType.isFeeDeduction() && isNotReversed();
```

Comment:
is reversal case already handled?

Reply:
yes, that was handled in SavingsAccountTransactionSummaryWrapper.java